pandas/io/common.py

```diff
@@ -162,13 +165,13 @@ def is_fsspec_url(url: FilePathOrBuffer) -> bool:
     )
 
 
-def get_filepath_or_buffer(
+def get_filepath_or_buffer(  # type: ignore[assignment]
```

Comment:
my local mypy needs that for line 170 and 172 but the CI mypy needs it apparently at that line (`TypeVar`s cannot have default values, could be fixed with `@overload`)